contents/handbook/help.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: How you can help
+sidebar: Handbook
+showTitle: true
+---
+
+## Getting yourself up and running - your first week
```

Comment:
Again, probably reference material. Also highlights that it's a little bit confusing who this guide is written for. a new employee? Existing employees?